nginx/index.html.md.erb

```diff
@@ -130,6 +130,62 @@ top of your `nginx.conf` file. You do not need to provide an
 <p class="note"><strong>Note</strong>: To name your modules directory something other than <code>modules</code>, use the NGINX <code>load_module</code> directive, providing a path to the module relative to the location of your <code>nginx.conf</code> file. For example:
 <code>load_module some_module_dir/my_module.so</code></p>
 
+
+### <a id='enabling-debugging'></a> Enabling Debugging ###
+Debugging helps to identify a bug in the program code if something goes wrong. It is generally used in developing or testing third-party or experimental modules.
+
+By default, debug logging will be disabled in the NGINX buildpack, which helps optimize performance. The user will need to set the debug level with the error_log directive in the nginx.conf to enable debug logging.
+
+To use the debug logging, add the `error_log` entry to your `nginx.conf` file follow the syntax below:
+
+```
+Syntax:	error_log file [level];
```

Comment:
Might be good to list what options are for the levels.

Reply:
I explain the levels here https://github.com/cloudfoundry/docs-buildpacks/blob/9d6b3d20801aca418dc680f4833824eafcc5a5ac/nginx/index.html.md.erb#L147. I can change it to show it as a list if you want